ltiauthenticator/lti11/auth.py

```diff
@@ -28,8 +37,33 @@ class LTI11Authenticator(Authenticator):
     auto_login = True
     login_service = "LTI 1.1"
 
+    config_description = Unicode(
+        default_value="JupyterHub LTI 1.1 external tool",
+        config=True,
+        help="""
+        The external tool's description.
+        """,
+    )
+
+    config_icon = Unicode(
+        default_value="",
+        config=True,
+        help="""
+        The icon used to represent the tool. This icon is usually displayed in the LTI 1.1 consumer interface.
+        The image should have 16x16 px, long-term https and only accepts the gif, png, or jpg file types.
+        """,
+    )
```

Comment:
I assume this is the unicode string to a public URL? Is that correct? Perhaps reasonable to be more explicit about that. I don't know if this is obvious or not given the context though, but I guess it could be useful to be explicit without knowing.

Reply:
Good point, I'll update the help content with a clarification on how to reference the icon image.